conformance_tests/test_tiovx/test_ext_supplementary_data.c

```diff
@@ -0,0 +1,1435 @@
+    ERROR_CHECK_VX_SUCCESS(tivxAddKernelTarget(general_kernel, TIVX_TARGET_HOST), NULL);;
+    ERROR_CHECK_VX_SUCCESS(vxAddParameterToKernel(general_kernel, 0, VX_INPUT, VX_TYPE_REFERENCE, VX_PARAMETER_STATE_REQUIRED), NULL);
+    ERROR_CHECK_VX_SUCCESS(vxAddParameterToKernel(general_kernel, 1, VX_OUTPUT, VX_TYPE_REFERENCE, VX_PARAMETER_STATE_REQUIRED), NULL);
+    ERROR_CHECK_VX_SUCCESS(vxFinalizeKernel(general_kernel), NULL);
+    /* end of section to remove */
+    target_test_kernel = tivxAddTargetKernel(MY_USER_TEST_TARGET_KERNEL, TIVX_TARGET_HOST, myUserTestTargetFunction, myGeneralKernelCreate, myGeneralKernelDelete, NULL, NULL);
+    general_test_kernel = vxAddUserKernel(context, "myUserTestTargetKernel", MY_USER_TEST_TARGET_KERNEL, NULL, 4, myGeneralKernelValidator, NULL, NULL);
+    EXPECT_VX_OBJECT(general_test_kernel, VX_TYPE_KERNEL);
+    ERROR_CHECK_VX_SUCCESS(tivxAddKernelTarget(general_test_kernel, TIVX_TARGET_HOST), NULL);;
+    ERROR_CHECK_VX_SUCCESS(vxAddParameterToKernel(general_test_kernel, 0, VX_INPUT, VX_TYPE_REFERENCE, VX_PARAMETER_STATE_REQUIRED), NULL);
+    ERROR_CHECK_VX_SUCCESS(vxAddParameterToKernel(general_test_kernel, 1, VX_INPUT, VX_TYPE_REFERENCE, VX_PARAMETER_STATE_REQUIRED), NULL);
+    ERROR_CHECK_VX_SUCCESS(vxAddParameterToKernel(general_test_kernel, 2, VX_INPUT, VX_TYPE_REFERENCE, VX_PARAMETER_STATE_REQUIRED), NULL);
+    ERROR_CHECK_VX_SUCCESS(vxAddParameterToKernel(general_test_kernel, 3, VX_OUTPUT, VX_TYPE_REFERENCE, VX_PARAMETER_STATE_REQUIRED), NULL);
+    ERROR_CHECK_VX_SUCCESS(vxFinalizeKernel(general_test_kernel), NULL);
+
+    user_test_kernel = vxAddUserKernel(context, "myUserTestKernel", MY_USER_TEST_KERNEL, myUserTestKernelFunction, 4, myUserTestKernelValidator, NULL, NULL);
+    ERROR_CHECK_VX_SUCCESS(vxAddParameterToKernel(user_test_kernel, 0, VX_INPUT, VX_TYPE_REFERENCE, VX_PARAMETER_STATE_REQUIRED), NULL);
+    ERROR_CHECK_VX_SUCCESS(vxAddParameterToKernel(user_test_kernel, 1, VX_INPUT, VX_TYPE_REFERENCE, VX_PARAMETER_STATE_REQUIRED), NULL);
+    ERROR_CHECK_VX_SUCCESS(vxAddParameterToKernel(user_test_kernel, 2, VX_OUTPUT, VX_TYPE_REFERENCE, VX_PARAMETER_STATE_REQUIRED), NULL);
+    ERROR_CHECK_VX_SUCCESS(vxAddParameterToKernel(user_test_kernel, 3, VX_OUTPUT, VX_TYPE_REFERENCE, VX_PARAMETER_STATE_REQUIRED), NULL);
+    ERROR_CHECK_VX_SUCCESS(vxFinalizeKernel(user_test_kernel), NULL);
+
+    my_dumb_copy_kernel = vxAddUserKernel(context, "MY_DUMB_IMAGE_COPY", MY_DUMB_IMAGE_COPY_KERNEL, myDumbImageCopyKernel, 2, myDumbImageCopyValidator, NULL, NULL);
+    ERROR_CHECK_VX_SUCCESS(vxAddParameterToKernel(my_dumb_copy_kernel, 0, VX_INPUT, VX_TYPE_REFERENCE, VX_PARAMETER_STATE_REQUIRED), NULL);
+    ERROR_CHECK_VX_SUCCESS(vxAddParameterToKernel(my_dumb_copy_kernel, 1, VX_OUTPUT, VX_TYPE_REFERENCE, VX_PARAMETER_STATE_REQUIRED), NULL);
+    ERROR_CHECK_VX_SUCCESS(vxFinalizeKernel(my_dumb_copy_kernel), NULL);
+}
+
+void tivxRemoveTargetKernelGeneral(void)
```

Comment:
Can you please change the name to "testRemoveTargetKernelGeneral"?  This will help us avoid namespace confusion

Reply:
done